### ISSUE TITLE
Default usage somehow ends up using old plugin and wrapper

### ISSUE THREAD

**Issue by jnorthrup** [takari/maven-wrapper#116]
![image](https://user-images.githubusercontent.com/73514/54262121-04418f00-45a0-11e9-9eab-4ed865a1eb8c.png)

**Comment by mosabua** [comment]
Somehow  you local network configuration does not seem to download the latest wrapper release and hence you are not getting the latest Maven version. I can not reproduce this on my setup and it seems isolated to your setup. 

Try to install the wrapper with mvn directly and see if that makes a difference. Beyond that check what you proxy config and such is..

**Comment by mosabua** [comment]
Wow ... just tried again and I do indeed get the problem. Not sure whats going on to be honest. I will update the docs to include the version.. then it should work. 

And in parallel I will try to figure out what is going on..

**Comment by mosabua** [comment]
Maven metadata in Central is correct. http://repo.maven.apache.org/maven2/io/takari/maven/maven-metadata.xml

**Comment by mosabua** [comment]
If you wipe the local Maven repo content in io/takari/maven it works correctly and loads the latest even without specifying. This might be a Maven bug..

**Comment by jnorthrup** [comment]
the updated README is probably perfect.  

i did some digging.  

something in the old lineage was sticky.

![image](https://user-images.githubusercontent.com/73514/54310378-d5a9cf80-4604-11e9-8663-b7e93c45a429.png)

now run from ./mvnw 0.4.2

![image](https://user-images.githubusercontent.com/73514/54310298-a6935e00-4604-11e9-85f6-8d196e0bef6b.png)

success? 
 
![image](https://user-images.githubusercontent.com/73514/54310738-a5166580-4605-11e9-8dfa-a2ff25510e20.png)

yes but.... evil lurks...


![image](https://user-images.githubusercontent.com/73514/54311246-c3c92c00-4606-11e9-83b3-b371b634e390.png)

 
```
jim@jim-Lenovo-ideapad-Y900-17ISK:~/kbrprime-pwa$ cat pom.xml 
[...]
    <repositories>
        <repository>
            <id>bintray-kotlin-kotlinx</id>
            <name>bintray</name>
            <url>https://kotlin.bintray.com/kotlinx</url>
        </repository>
    </repositories>
```

does the new README solve things for those who like the  "in anger" programming books ?

![image](https://user-images.githubusercontent.com/73514/54311445-29b5b380-4607-11e9-87be-0cfb9bcdc4bb.png)

absolutely!  a two button maven wrapper fix for maven-3.6, great job!
